EntAppProject/src/main/java/com/jobook/EntAppProject/dto/UserDTO.java

```diff
@@ -0,0 +1,69 @@
+package com.jobook.EntAppProject.dto;
+
+import com.jobook.EntAppProject.model.Address;
+
+public class UserDTO {

```

Comment:
If it's already in a package named dto, consider dropping DTO from the class name, and just call it user.  The package gives it context.